src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1780,7 +1808,7 @@ private Conversion ClassifyImplicitNullableConversion(TypeSymbol source, TypeSym
         }
 
         private delegate Conversion ClassifyConversionFromExpressionDelegate(ConversionsBase conversions, BoundExpression sourceExpression, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics, bool arg);
```

Comment:
Should this also use `TypeSymbolWithAnnotations`?

Reply:
Yes, it should. However it looks like `GetTupleLiteralConversion` is not being called currently from `NullableWalker`. I've added a couple of skipped tests for now.

---
In reply to: [164205895](https://github.com/dotnet/roslyn/pull/23981#discussion_r164205895) [](ancestors = 164205895)